backend/sh/copy_file_to_image.sh

```diff
@@ -61,7 +61,7 @@ if [ "${p}" != "boot" ]; then
 else
     if [ ! -d $(dirname ${o}) ]; then
         d_echo "mmd -D sS -i /tmp/${uuid}/${p}.img ::$(dirname ${o})"
-        mmd -D sS -i /tmp/${uuid}/${p}.img ::$(dirname ${o})
+        mmd -D sS -i /tmp/${uuid}/${p}.img ::$(dirname ${o}) || true
```

Comment:
doesnt this skip the error in every case?

Reply:
It is skipping it but returns exit code 1. This leads to fail in our api.

Reply:
what do you mean by our api?

e.g.
```
! true || true; echo $?
```
will alwas print 0